docs/pom.xml

```diff
@@ -297,25 +297,68 @@
                 </executions>
             </plugin>
             <plugin>
-                <!--
-                    Override the javadoc configuration to include source jars from our dependencies in order to build combined
-                    documentation for the entire distribution.
-                -->
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>unpack</id>
+                        <goals>
+                            <goal>unpack-dependencies</goal>
+                        </goals>
+                        <phase>package</phase>
+                        <configuration>
+                            <classifier>sources</classifier>
+                            <includeGroupIds>gov.nsa.datawave, gov.nsa.datawave.webservices</includeGroupIds>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
                 <artifactId>maven-javadoc-plugin</artifactId>
+                <version>3.0.1</version>
                 <executions>
                     <execution>
-                        <id>module-apidocs</id>
+                        <id>package</id>
                         <goals>
                             <goal>jar</goal>
                         </goals>
                         <phase>package</phase>
-                        <configuration combine.children="append">
-                            <includeDependencySources>true</includeDependencySources>
-                            <includeTransitiveDependencySources>true</includeTransitiveDependencySources>
-                            <dependencySourceIncludes>
-                                <dependencySourceInclude>gov.nsa.datawave:*</dependencySourceInclude>
-                                <dependencySourceInclude>gov.nsa.datawave.webservices:*</dependencySourceInclude>
-                            </dependencySourceIncludes>
+                        <configuration>
```

Comment:
This can't be inherited from the parent? Previously, it was using `combine.children="append"` (and the same execution ID as in the parent (`module-apidocs`)) to add configuration to that specified in the parent. Now the configuration is copied from the parent.